CHAOS-4162 ChaosHub changes (#5326)
harness/developer-hub: docs/chaos-engineering/configure-chaos-experiments/chaos-hubs/add-chaos-hub.md

---
title: Add a custom ChaosHub
sidebar_position: 2

> description: Steps to add a ChaosHub to your project

Reviewer: ```suggestion
description: Steps to add a custom ChaosHub to your project
```

---

> This sections describes reasons for adding a custom ChaosHub and how to add and connect a custom ChaosHub. 

Reviewer: ```suggestion
This section describes the reasons for adding a custom ChaosHub and how to add a custom ChaosHub. 
```

* Provide secure access to your custom experiments.

## Prerequisites to add a custom ChaosHub

> 1. Enure you have a Git repository for your custom ChaosHub, where you will store experiments and faults. The repository must include two folders: `experiments` and `faults`. Here's an example repo:

Reviewer: ```suggestion
1. Ensure you have a Git repository for your custom ChaosHub, where you will store experiments and faults. The repository must include two folders: `experiments` and `faults`. Here's an example repo:
```
